src/components/offering/label.tsx

```diff
@@ -0,0 +1,11 @@
+import { colors } from '@hedviginsurance/brand'
+import styled from 'react-emotion'
+
+/* TODO: rename */
```

Comment:
Good idea, i think `Label` is easily confused with the html element `label`, which is tightly coupled with form input elements

Reply:
Label was dead code unfortunately and is now removed